site/frontend/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "lib": ["es5", "es2015", "dom"],
+    "target": "es5",
```

Comment:
The PR description says ES2015, but this is targeting ES5. I would suggest targeting _ES2017_ or newer, so you avoid the regenerator cruft for async functions, which makes them annoying to debug.

This shouldn't lose support for any reasonable browser (at $dayjob we target ES2020). Likely any browser that does not support ES2017 today is actively insecure to expose to the internet.

Reply:
Yeah so actually, this file only exists for type checking (npm run check). Parcel by itself doesn't do any typechecking and doesn't generate the regenerator polyfill, I checked. Maybe I should put this into README.md.

Reply:
Ah, I should've noticed tsc was only used there.

Putting some basic docs about the build process in the readme would be useful, yeah